conda.recipe/meta.yaml

```diff
@@ -15,31 +15,40 @@ source:
 requirements:
   build:
```

Comment:
The way we do this in zipline is we load up the setup.py and pull the reqs out that way:

https://github.com/quantopian/zipline/blob/master/conda/zipline/meta.yaml
https://github.com/quantopian/zipline/blob/master/setup.py